whisper_online_server.py

```diff
@@ -28,35 +34,25 @@
 asr, online = asr_factory(args)
 min_chunk = args.min_chunk_size
 
-
-if args.buffer_trimming == "sentence":
-    tokenizer = create_tokenizer(tgt_language)
-else:
-    tokenizer = None
-online = OnlineASRProcessor(asr,tokenizer,buffer_trimming=(args.buffer_trimming, args.buffer_trimming_sec))
-
 # warm up the ASR because the very first transcribe takes more time than the others. 
 # Test results in https://github.com/ufal/whisper_streaming/pull/81
 msg = "Whisper is not warmed up. The first chunk processing may take longer."
 if args.warmup_file:
     if os.path.isfile(args.warmup_file):
         a = load_audio_chunk(args.warmup_file,0,1)
         asr.transcribe(a)
-        print("INFO: Whisper is warmed up.",file=sys.stderr)
+        logger.info("Whisper is warmed up.")
     else:
-        print("WARNING: The warm up file is not available. "+msg,file=sys.stderr)
+        logger.warning("The warm up file is not available. "+msg)
 else:
-    print("WARNING: " + msg, file=sys.stderr)
+    logger.warning(msg)
```

Comment:
Not sure this warrants a warning, as I think not many people will have a warmup file laying around? (I could be wrong though...)

Reply:
Bit of a subjective call here, I guess - is "this isn't going to be as fast as it could be" a warning condition? Looking at the whole block I actually think line 46 should be a hard fail, because the user has explicitly requested something that doesn't exist.

Reply:
ok, then make the hard fail here